web/src/components/Diagram/components/DAG/Controls.tsx

```diff
@@ -5,23 +5,31 @@ import AffectedSpanControls from './AffectedSpanControls';
 
 interface IProps {
   isMiniMapActive: boolean;
+  isTimeline?: boolean;
   onMiniMapToggle(): void;
 }
 
-const Controls = ({isMiniMapActive, onMiniMapToggle}: IProps) => {
+const Controls = ({isMiniMapActive, onMiniMapToggle, isTimeline = false}: IProps) => {
   const {fitView, zoomIn, zoomOut} = useReactFlow();
 
   return (
     <>
-      <S.SelectorControls>
+      <S.SelectorControls $isTimeline={isTimeline}>
         <AffectedSpanControls />
       </S.SelectorControls>
-      <S.Controls>
-        <S.ZoomButton icon={<ZoomInOutlined />} onClick={() => zoomIn()} type="text" />
-        <S.ZoomButton icon={<ZoomOutOutlined />} onClick={() => zoomOut()} type="text" />
-        <S.ZoomButton icon={<ExpandOutlined />} onClick={() => fitView()} type="text" />
-        <S.ZoomButton icon={<ApartmentOutlined />} onClick={onMiniMapToggle} type="text" $isActive={isMiniMapActive} />
-      </S.Controls>
+      {!isTimeline && (
```

Comment:
This is a small thing but I think is preferable to evaluate positive cases like isDag instead of negative ones. We can have that flag be true as default

Reply:
ok! nice